src/services/albums-service.js

```diff
@@ -0,0 +1,23 @@
+import axios from 'axios';
+
+const ALBUMS_API = 'http://localhost:8080/albums';
```

Comment:
This will need to switch to the env variable || localhost once we deploy to production